Chapter-2-modules-separation/Src/Offers/Fitnet.Offers.Api/Prepare/PassExpiredEventHandler.cs

```diff
@@ -7,29 +7,18 @@ namespace EvolutionaryArchitecture.Fitnet.Offers.Api.Prepare;
 using DataAccess.Database;
 using Passes.IntegrationEvents;
 
-internal sealed class PassExpiredEventHandler : IIntegrationEventHandler<PassExpiredEvent>
+internal sealed class PassExpiredEventHandler(
+    IEventBus eventBus,
+    OffersPersistence persistence,
+    ISystemClock systemClock) : IIntegrationEventHandler<PassExpiredEvent>
```

Comment:
I will create issue to remove ISystemClock because now dotnet has native interfaces for that purpose